### ISSUE TITLE
Grid elements have a tab index of -1

### ISSUE THREAD

**Issue by carlicockrell** [grabs/moodle-mod_unilabel#52]
We have noticed that you cannot see a focus indicator when you tab to elements that are created using grid. This is because the grid element has a tab index of -1. In order to fix this, we had to create a pseudoclass so we could ensure that the grid components have a tabindex=0. We wanted to let you know as others may report this to you in the future, as well. Here's more about how tabindex functions: https://www.a11y-collective.com/blog/tabindex-accessibility/

**Comment by grabs** [comment]
Hi Carli,

there is no tab index of “-1”. But actually the tile was not highlighted when it got the focus. This is now fixed and the focus is now visible.

Best regards
Andreas
